### PR TITLE
I guess I will make a lot of changes ^_^

### PR DESCRIPTION
**Pull request by MrNopi**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +17,30 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arr = new ArrayList<>();
+
+        File file = new File(fileName);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to create a solution without this variable

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +17,30 @@
...
+        if (file.length() == 0) {
+            return new String[]{};
+        }
+        Path path = Paths.get(fileName);
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: same

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -8,7 +17,30 @@
...
+        try {
+            List<String> array = Files.readAllLines(path);
+            for (String value : array) {
+                String[] tmp = value.toLowerCase().split(" ");
+                for (String s : tmp) {
+                    if (s.startsWith("w")) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: make "w" constant of a class

Comment (MrNopi) [reply src/main/java/core/basesyntax/FileWork.java]: > 
> 
> make "w" constant of a class

I will also make regex constant, is it fine?

Comment (Tarabarnik) [reply src/main/java/core/basesyntax/FileWork.java]: as you wish :) it's optional

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (MrNopi) [reply src/main/java/core/basesyntax/FileWork.java]: Which one? Without File or without list?

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -1,5 +1,11 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: we shouldn't use multiple imports like `import java.util.*;`

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (Tarabarnik) [reply src/main/java/core/basesyntax/FileWork.java]: what about this comment?

File: src/main/java/core/basesyntax/FileWork.java [hunk 5]
```diff
@@ -8,7 +14,29 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String W = "w";
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: will this name of constant be informative, if you change its value later?

File: src/main/java/core/basesyntax/FileWork.java [hunk 6]
```diff
@@ -8,7 +15,27 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    private static final String START_OF_WORDS = "w";
+    private static final String REGEX_NON_WORDS = "[^a-z+]";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> arr = new ArrayList<>();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: variable names should be full and informative. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.5-non-constant-field-names

**Pull request closed without merging**